components/org.wso2.carbon.business.rules.core/src/main/java/org/wso2/carbon/business/rules/core/services/TemplateManagerService.java

```diff
@@ -1481,4 +1493,40 @@ private String removeCRLFCharacters(String str) {
         return str;
     }
 
+    private void loadAndSaveAnalyticsSolutions(String solutionType) throws TemplateManagerServiceException {
+        if (Objects.nonNull(availableTemplateGroups.get(solutionType))) {
```

Comment:
Simply use `availableTemplateGroups.get(solutionType) != null` in this kind of scenarios. Purpose of Objects.nonNull is to use is as predicates (`filter(Objects::isNull)`).

Reply:
Fixed by b14dd18f396f799bd32670842869feb2475e778e